api/v1/controllers/AuthController.js

```diff
@@ -0,0 +1,68 @@
+var Promise = require('bluebird');
+
+var errors = require('../errors');
+var AuthService = require('../services/AuthService');
+var UserService = require('../services/UserService');
+
+var router = require('express').Router();
+
+function _issueByEmail(email, password) {
+	if (!password && password !== null) {
+		throw new TypeError("To issue a token by email without checking the " +
+			"user's password, you must pass the password as the null type");
```

Comment:
Instead of forcing consumers of this function to pass null, make password an optional arg. I.e function foo (b) { b = b || 0;} (or function foo (b = 0) {} if using ES6)